extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosInterface.java

```diff
@@ -260,9 +260,13 @@ <T> T makePostRequest(String url, Optional<Map<String, String>> parameters,
    * Note that making this a separate method to avoid polluting throw lists.
    */
   private void maybeRethrowAsUploadError(HttpResponseException e) throws UploadErrorException {
```

Comment:
optional nit: would be helpful to pull these strings out to consts and document them, or give them self-documenting names like `GOOG_MISMATCH_API_ERROR_MSG` and `GOOG_MISMATCH_API_ERROR_OLD_MSG`

Reply:
Thanks! Updated.